src/index.ts

```diff
@@ -13,9 +15,15 @@ export function sortByDesc<T = any>(values: any, iteratees?: any): T[] {
     return sort(values).desc(iteratees);
 }
 
-export function orderBy<T = any>(values: T[], iteratees: string | string[], orders: string | string[]): T[] {
+export function orderBy<T = any>(
+    values: T[],
+    iteratees: CallbackFunctionVariadicAnyReturn | string | CallbackFunctionVariadicAnyReturn[] | string[],
```

Comment:
What would be an example of `CallbackFunctionVariadicAnyReturn[]` as use-case for sorting?

Reply:
On second thought this is probably unneeded, since all sorting logic can be passed in one function. I'll remove that asap.